lib/PromptSession.ts

```diff
@@ -22,7 +22,6 @@ export class PromptSession {
 
 		// tslint:disable:object-literal-sort-keys
 		if (config != null && !config.project.isShowcase) {
-			//throw new Error("Add command is supported only on existing project created with igntie-ui-cli");
 			projLibrary = this.templateManager.getProjectLibrary(config.project.framework, config.project.projectType);
```

Comment:
So no change here. Not sure this functionality should remain, though we can iron this out later on.